### PR TITLE
Feature/se 418/amend orders test for new filtering

### PR DESCRIPTION
**Pull request by cscialino**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

- Made amendments to "Orders.cs" to meet new syntax for filtering
- Changes Scope to Id. Scope, Code to Id.Code, OrderBook to OrderBookId.Code, etc. 
- Amended "in" filtering to include brackets
- Changed filtering for porperties, adding full key

Tests not passed, awaiting for new API features to be updated

### CODE REVIEW

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs [hunk 1]
```diff
@@ -321,34 +320,35 @@ public void Upsert_And_Retrieve_Simple_Orders()
 
             var quantityFilter = _ordersApi.ListOrders(asAt:
                 t,
-                filter: $"Quantity gt 100 and Scope eq '{testScope}' and Id in '{order1Filter}', '{order2Filter}', '{order3Filter}'");
+                filter: $"Quantity gt 100 and Id.Scope eq '{testScope}' and Id.Code in ('{order1Filter}', '{order2Filter}', '{order3Filter}')");
```
Comment (adp-finbourne) [inline sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:323]: `order1Filter` etc  look like they're in the from "{Scope}/{Code}"; if we only want to filter by Id.Code, the filters will need to be changed to "{Code}"

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs [hunk 2]
```diff
@@ -321,34 +320,35 @@ public void Upsert_And_Retrieve_Simple_Orders()
...
             Assert.That(instrumentFilter.Values.All(rl => rl.LusidInstrumentId.Equals(_instrumentIds[0])));
 
-            var sideFilter = _ordersApi.ListOrders(asAt: t, filter: $"Side eq 'Sell' and Scope eq '{testScope}'");
+            var sideFilter = _ordersApi.ListOrders(asAt: t, filter: $"Side eq 'Sell' and Id.Scope eq '{testScope}'");
 
             Assert.That(sideFilter.Values.Count, Is.EqualTo(1));
             Assert.That(sideFilter.Values.All(rl => rl.Side.Equals("Sell")));
 
-            var orderBookFilter = _ordersApi.ListOrders(asAt: t, filter: $"OrderBook eq '{testScope}/AnotherOrdersTestBook'");
+            var orderBookFilter = _ordersApi.ListOrders(asAt: t, filter: $"OrderBookId.Code eq '{testScope}'");
```
Comment (adp-finbourne) [inline sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:347]: This should be the Code `AnotherOrdersTestBook` rather than the Scope `{testScope}`

**Review by adp-finbourne — CHANGES_REQUESTED** [review]

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs (hunk 1 above)
Comment (cscialino) [reply sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:323]: Amended to meet above inputs:
> var order1Filter = $"{testScope}/{order1}"; --> var order1Filter = $"{order1}";

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs (hunk 2 above)
Comment (cscialino) [reply sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs]: Amended filtering parameter to: `AnotherOrdersTestBook `

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs [hunk 3]
```diff
@@ -314,9 +314,9 @@ public void Upsert_And_Retrieve_Simple_Orders()
 
             var t = upsertResult.Values.First().Version.AsAtDate;
 
-            var order1Filter = $"{testScope}/{order1}";
-            var order2Filter = $"{testScope}/{order2}";
-            var order3Filter = $"{testScope}/{order3}";
+            var order1Filter = $"{order1}";
+            var order2Filter = $"{order2}";
+            var order3Filter = $"{order3}";
```
Comment (adp-finbourne) [inline sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:319]: Given that `order1` etc are already strings, we could just pass them directly into the filter on line 323 - might be a bit clearer?

**Review by adp-finbourne — CHANGES_REQUESTED** [review]

**Pull request merged**